solr/solr-ref-guide/modules/deployment-guide/pages/circuit-breakers.adoc

```diff
@@ -24,12 +24,19 @@ resource configuration.
 Circuit breakers should be used when the user wishes to trade request throughput for a higher Solr stability.
 If circuit breakers are enabled, requests may be rejected under the condition of high node duress with HTTP error code 429 'Too Many Requests'.
 
-It is up to the client to handle this error and potentially build a retrial logic as this should ideally be a transient situation.
+It is up to the client to handle this error and potentially build retry logic as this should be a transient situation.
+
+In a sharded collection, when a circuit breaker trips on one shard, the entire query will fail,
+even if the other shard requests succeed. This will multiply the failures seen by the end users.
+Setting the `shards.tolerant=true` parameter on requests can help with graceful degradation when
+circuit breaker thresholds are reached on some nodes. See the <<shards.tolerant Parameter>> for details.
 
 == Circuit Breaker Configurations
 All circuit breaker configurations are listed as independent `<circuitBreaker>` entries in `solrconfig.xml` as shown below.
 A circuit breaker can register itself to trip for query requests and/or update requests. By default only search requests are affected. A user may register multiple circuit breakers of the same type with different thresholds for each request type.
 
+Configuring circuit breakers for update requests in a collection with NRT replicas can result in
+inconsistent contents in different replicas.
```

Comment:
Sorry, thought I removed that and asked the question in this comment string. I'll pull it out. I don't have a super-deep understanding of NRT, but I think the replicas pull all outstanding updates from the tlog using real-time get, so that should be safe for lost update notifications. If it actually works that way...

Reply:
I put up a PR for SOLR-16982: https://github.com/apache/solr/pull/1930 so that should deal with this concern